public/assetlinks/assetlinks.debug.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏i wonder if it could be possible to move these files into our env configuration repo. 
Then to read these file and inject it inside the container.
Two possible solution: 
- inject the files via a docker volumes to set them as expected by the container. -v /local/public/assetlinks/:/dist/public/assetlinks/
- pass the content of the json as env var in the docker and then gen_conf.py will read the file and create the env var